### PR TITLE
add vmdk test workflow

### PR DESCRIPTION
**Pull request by yih-redhat**

### CODE REVIEW

**Comment by henrywang** [comment]
`commit lint` test failed. Please change your commit to `ci: add vmdk test workflow`. Thanks.

**Comment by yih-redhat** [comment]
> `commit lint` test failed. Please change your commit to `ci: add vmdk test workflow`. Thanks.

Done.

File: .github/workflows/centos-stream-9.yml [hunk 1]
```diff
@@ -690,3 +690,88 @@ jobs:
           path: |
             *.json
             *.log
+
+  pre-edge-vsphere-cs9:
+    needs: pr-info
+    if: ${{ needs.pr-info.outputs.allowed_user == 'true' && github.event.issue.pull_request &&
+            (endsWith(github.event.comment.body, '/test-cs9') ||
+            endsWith(github.event.comment.body, '/test-cs9-vsphere')) }}
```
Comment (henrywang) [inline .github/workflows/centos-stream-9.yml:698]: Please add `/test-cs9-vsphere` at the start of this workflow.

File: .github/workflows/rhel-9-3.yml [hunk 2]
```diff
@@ -782,3 +782,88 @@ jobs:
           path: |
             *.json
             *.log
+
+  pre-edge-vsphere-93:
+    needs: pr-info
+    if: ${{ needs.pr-info.outputs.allowed_user == 'true' && github.event.issue.pull_request &&
+            (endsWith(github.event.comment.body, '/test-rhel-9-3') ||
+            endsWith(github.event.comment.body, '/test-rhel-9-3-vsphere')) }}
```
Comment (henrywang) [inline .github/workflows/rhel-9-3.yml:790]: Please add `/test-rhel-9-3-vsphere` at the start of this workflow.

File: .github/workflows/centos-stream-9.yml [hunk 3]
```diff
@@ -690,3 +690,88 @@ jobs:
...
+          description: 'Runner has been deploying...'
+          target_url: 'https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  edge-vsphere-cs9:
+    needs: [pr-info, pre-edge-vsphere-cs9]
+    runs-on: [kite, x86_64, rhos-01, centos-stream-9, medium]
+    env:
+      STATUS_NAME: edge-vsphere-installer-cs9
```
Comment (henrywang) [inline .github/workflows/centos-stream-9.yml:719]: What does the `installer` mean in STATUS_NAME. I don't think it's reasonable.

File: .github/workflows/rhel-9-3.yml [hunk 4]
```diff
@@ -782,3 +782,88 @@ jobs:
...
+    runs-on: ubuntu-latest
+    env:
+      STATUS_NAME: edge-vsphere-installer-9.3
```
Comment (henrywang) [inline .github/workflows/rhel-9-3.yml:793]: What does the `installer` mean in STATUS_NAME. I don't think it's reasonable.

File: .github/workflows/rhel-9-3.yml [hunk 5]
```diff
@@ -782,3 +782,88 @@ jobs:
...
+          description: 'Runner has been deploying...'
+          target_url: 'https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  edge-vsphere-93:
+    needs: [pr-info, pre-edge-vsphere-93]
+    runs-on: [kite, x86_64, rhos-01, rhel-9-3, large]
+    env:
+      STATUS_NAME: edge-vsphere-installer-9.3
```
Comment (henrywang) [inline .github/workflows/rhel-9-3.yml:811]: What does the `installer` mean in STATUS_NAME. I don't think it's reasonable.

File: .github/workflows/centos-stream-9.yml [hunk 6]
```diff
@@ -690,3 +690,88 @@ jobs:
...
             *.log
+
+  pre-edge-vsphere-cs9:
+    needs: pr-info
+    if: ${{ needs.pr-info.outputs.allowed_user == 'true' && github.event.issue.pull_request &&
+            (endsWith(github.event.comment.body, '/test-cs9') ||
+            endsWith(github.event.comment.body, '/test-cs9-vsphere')) }}
+    runs-on: ubuntu-latest
+    env:
+      STATUS_NAME: edge-vsphere-installer-cs9
```
Comment (henrywang) [inline .github/workflows/centos-stream-9.yml:701]: What does the `installer` mean in STATUS_NAME. I don't think it's reasonable.

**Review by henrywang — CHANGES_REQUESTED** [review]

File: .github/workflows/centos-stream-9.yml (hunk 1 above)
Comment (yih-redhat) [reply .github/workflows/centos-stream-9.yml:699]: added

File: .github/workflows/rhel-9-3.yml (hunk 2 above)
Comment (yih-redhat) [reply .github/workflows/rhel-9-3.yml:791]: added

File: .github/workflows/centos-stream-9.yml (hunk 3 above)
Comment (yih-redhat) [reply .github/workflows/centos-stream-9.yml]: removed installer

File: .github/workflows/rhel-9-3.yml (hunk 4 above)
Comment (yih-redhat) [reply .github/workflows/rhel-9-3.yml]: removed installer

File: .github/workflows/rhel-9-3.yml (hunk 5 above)
Comment (yih-redhat) [reply .github/workflows/rhel-9-3.yml]: removed installer

File: .github/workflows/centos-stream-9.yml (hunk 6 above)
Comment (yih-redhat) [reply .github/workflows/centos-stream-9.yml]: removed installer

**Review by henrywang — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**